selectors.yml

```diff
@@ -373,4 +373,16 @@ selectors:
     name: "private-testnet-mica"
   48900:
     selector: 17198166215261833993
-    name: "ethereum-mainnet-zircuit-1"
\ No newline at end of file
+    name: "ethereum-mainnet-zircuit-1"
+  223:
+    selector: 5406759801798337480
+    name: "bitcoin-mainnet-bsquared-1"
+  200901:
+    selector: 7937294810946806131
+    name: "bitcoin-mainnet-bitlayer-1"
+  60808:
+    selector: 3849287863852499584
+    name: "ethereum-mainnet-sepolia-bob-1"
+  1329:
+    selector: 9027416829622342829
+    name: "sei-mainnet-pacific"
```

Comment:
Why pacific?

Reply:
I assumed it had to be there, https://www.docs.sei.io/dev-chains#mainnet, if not I'll remove it. 
for reference, for testnet is `sei-testnet-atlantic`.

Reply:
Ah yes you're right. LGTM.

Reply:
We usually don't name mainnets, unless there are more than one.
Testnets is more frequent to have multiple, that's why it's more common for them to be named, although there are quite a few unnamed ones (e.g. `binance_chain-testnet`).
Unless you have a strong hint there'll be multiple mainnets in this project in the future, I'd prefer if it was omitted, and kept only for testnets, which are commonly more volatile.

Reply:
@andrevmatos beat me to it, +1, they'd kill testnets during migration, but mainnet is almost always a hardfork

Reply:
@andrevmatos / @matYang  Hmm, I initially also thought this looks weird but going by the guide here (https://smartcontract-it.atlassian.net/wiki/spaces/DEPLOY/pages/599491447/Network+naming+proposal)  wouldn't the network name always be added in the name? Maybe it just happens that we didn't have mainnets with a specific name before, but according to the guide, I haven't seen an exception for mainnets?

Reply:
Most mainnets do have a name, e.g. ethereum-mainnet is historically called Homestead. We simply never added it, because each project usually only has one mainnet. This is not explicit in the Network naming proposal, yes, but was in place before we started this repo, and we just played along, and now should stick to it for consistency.
This can/should be expressed in the documentation though.

Reply:
who owns the guide? we should look into updating that

Reply:
@matYang @andrevmatos 
Not sure on who owns the guide but agree that if needed let's update it.
Also exploring this option, given that they listed the mainnet name as pacific-1 (https://www.docs.sei.io/dev-chains) , should we be concerned that there might be a pacific-2? And if so, and we went ahead with removing the mainnet name, would we be able to change later? I'm asking because we have no solid info on future plans for this chain but just based on their approach and naming.

Reply:
We wouldn't/shouldn't change existing names, as there may be products depending on them, but we'll then be able to add a `sei-mainnet-pacific-2` or something like that; not pretty, but alas, we're not in charge of that project.

Either way, I don't think we need to worry about that just right now, or deviate from our established pattern for future uncertainties which aren't clear when or if will take place.

Reply:
Sounds good then, let's stick with the current convention.

Reply:
thanks all, changed from `sei-mainnet-pacific` to `sei-mainnet`.